exporter/exporterhelper/exporterhelperprofiles/profiles_batch_test.go

```diff
@@ -172,3 +158,13 @@ func (req *tracesRequest) Export(ctx context.Context) error {
 func (req *tracesRequest) ItemsCount() int {
 	return req.td.SpanCount()
 }
+
+func (req *tracesRequest) Merge(_ context.Context, _ exporterhelper.BatchRequest) (exporterhelper.BatchRequest, error) {
+	return nil, nil
+}
+
+// MergeSplit splits and/or merges the profiles into multiple requests based on the MaxSizeConfig.
+func (req *tracesRequest) MergeSplit(_ context.Context, _ exporterbatcher.MaxSizeConfig, _ exporterhelper.BatchRequest) (
+	[]exporterhelper.BatchRequest, error) {
+	return nil, nil
+}
```

Comment:
Is there a test where `tracesRequest` needs the `BatchRequest` implementation?

Reply:
I agree this is confusing. tracesRequest in this file is solely for testing that merging profilesBatchRequest and tracesRequest would fail. Let me rename it to dummyRequest.